lib/improver/cli/remainder.py

```diff
@@ -0,0 +1,151 @@
+                     "the requested operation. In this case it needs to "
+                     "subtract the cubes from a cube of ones to find the "
+                     "remainder by using the operation - ")
+    parser.add_argument("input_filenames", metavar="INPUT_FILENAMES",
+                        nargs="+", type=str,
+                        help="Paths to the input NetCDF files. Each input"
+                        " file should be able to be loaded as a single "
+                        " iris.cube.Cube instance. The resulting file"
+                        " metadata will be based on the first file but"
+                        " its metadata can be overwritten via"
+                        " the metadata_jsonfile option.")
+    parser.add_argument("output_filepath", metavar="OUTPUT_FILE",
+                        help="The output path for the processed NetCDF.")
+    parser.add_argument("--operation", metavar="OPERATION",
+                        default="-",
+                        choices=["-", "subtract"],
+                        help="Operation to use in combining NetCDF datasets"
+                        " Default=- i.e. subtract ", type=str)
+    parser.add_argument("--new-name", metavar="NEW_NAME",
+                        default=None,
+                        help="New name for the resulting dataset. Will"
+                        " default to the name of the first dataset if "
+                        "not set.", type=str)
+    parser.add_argument("--metadata_jsonfile", metavar="METADATA_JSONFILE",
+                        default=None,
+                        help="Filename for the json file containing "
+                        "information for bounds expansion.", type=str)
+    parser.add_argument('--warnings_on', action='store_true',
+                        help="If warnings_on is set (i.e. True), "
+                        "Warning messages where metadata do not match "
+                        "will be given. Default=False", default=False)
+
+    args = parser.parse_args(args=argv)
+
+    new_metadata = cli.inputjson(args.metadata_jsonfile)
+    # Load cubes
+    cubelist = []
+    new_cube_name = args.new_name
+    for filename in args.input_filenames:
+        new_cube = cli.inputcube(filename)
+        cubelist.append(new_cube)
+        if new_cube_name is None:
+            new_cube_name = new_cube.name()
+        if args.warnings_on:
+            if (args.new_name is None and
+                    new_cube_name != new_cube.name()):
+                msg = ("Defaulting to first cube name, {} but combining with"
+                       "a cube with name, {}.".format(
+                            new_cube_name, new_cube.name()))
+                warnings.warn(msg)
+    cube_of_ones = cubelist[0].copy()
+    cube_of_ones.data = np.ones((cube_of_ones.shape), dtype="float32")
+    cubelist.insert(0, cube_of_ones)
+    # Process Cube
+    process(*cubelist, operation=args.operation, new_name=new_cube_name,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected keyword argument 'output' in function call](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4606176)